src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -24,37 +36,265 @@ const ActorInit En_Holl_InitVars = {
     (ActorFunc)EnHoll_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_8089A5D0[] = {
+static UNK_TYPE D_8089A550[] = { 0x465055F0, 0x00000000, 0x08000800, 0xFFFFFFFF, 0xB9B055F0, 0x00000000,
+                                 0x00000800, 0xFFFFFFFF, 0xB9B0AA10, 0x00000000, 0x00000000, 0xFFFFFFFF,
+                                 0x4650AA10, 0x00000000, 0x08000000, 0xFFFFFFFF };
+
+static UNK_TYPE D_8089A590[] = { 0xD7000000,  0xFFFFFFFF, 0xFCFFFFFF, 0xFFFDF638, 0x01004008,
+                                 &D_8089A550, 0x06000204, 0x00000406, 0xDF000000, 0x00000000 };
+
+static EnHoll* sThis = NULL;
```

Comment:
I'm not a fan of this name :s

Reply:
Do you have any other suggestions? The only two alternatives I came up with were sThisPtr and sSelfPtr, but putting the word Ptr in seemed redundant, adn we are currently using "this" instead of "self", so I figured sThis was best.

Reply:
Unless I'm missing something, this just looks like it is tracking which loading plane should make a sound. So maybe something like `sSoundLoadingPlane` or `sNoiseLoadingPlane`.

Reply:
Fixed in https://github.com/zeldaret/mm/pull/270/commits/603adc9b2da369246cb8019a52363bcee4efa3f5, so I'm marking this as resolved.